rpcs3/Emu/NP/rpcn_config.cpp

```diff
@@ -106,6 +106,255 @@ std::vector<std::pair<std::string, std::string>> cfg_rpcn::get_hosts()
 	return vec_hosts;
 }
 
+std::string cfg_rpcn::get_country() const
+{
+	return country.to_string();
+}
+
+std::vector<cfg_rpcn::country_code> cfg_rpcn::get_countries()
+{
+	std::vector<country_code> countries;
+	countries.push_back({"United States", {'u','s'}});
+	countries.push_back({"Japan", {'j', 'p'}});
+	countries.push_back({"Afghanistan", {'a', 'f'}});
+	countries.push_back({"Albania", {'a', 'l'}});
+	countries.push_back({"Algeria", {'d', 'z'}});
+	countries.push_back({"American Samoa", {'a', 's'}});
+	countries.push_back({"&orra", {'a', 'd'}});
```

Comment:
Andorra

Reply:
Sorry i missed that when i was shortening the names.